crates/router/src/connector/stripe/transformers.rs

```diff
@@ -940,7 +940,7 @@ fn get_bank_debit_data(
         } => {
             let bacs_data = BankDebitData::Bacs {
                 account_number: account_number.to_owned(),
-                sort_code: sort_code.to_owned(),
+                sort_code: Secret::new(sort_code.clone().expose().replace('-', "")),
```

Comment:
is this "-" sent from SDK?

Reply:
Yes, SDK sends sort code in xx-xx-xx format